docs/kubectl_patch.md

```diff
@@ -25,7 +25,7 @@ kubectl patch node k8s-node-1 -p '{"spec":{"unschedulable":true}}'
 
 ```
   -h, --help=false: help for patch
-  -p, --patch="": The patch to be applied to the resource JSON file.
+  -p, --patch="": The patch to be appied to the resource JSON file.
```

Comment:
yea, a typo but not by this PR, and brendandburns has fixed this small typo. I'll rebase onto master.